charts/rollups-validator-node/ci/test-values.yaml.tpl

```diff
@@ -64,11 +56,29 @@ extraDeploy:
       namespace: "{{ .Release.Namespace }}"
       name: "{{ .Release.Name }}-inspect-server"
     data:
-      SESSION_ID: "default_rollups_id"
+      SESSION_ID: "default_session_id"
       INSPECT_SERVER_ADDRESS: "0.0.0.0:5005"
       SERVER_MANAGER_ADDRESS: '{{ include "validator.fullname" . }}-server-manager:5001'
 
 validator:
+  localnode:
+    enabled: true
+    anvil:
+      enabled: true
+      image:
+        registry: sunodo
+        repository: anvil
+        tag: 2.0.0
+      anvilState:
+        image:
+          registry: sunodo
+          repository: rollups-node
+          tag: 0.2.0
+    deployer:
+      image:
+        registry: oap75
+        repository: rollups-cli
+        tag: pr-827
```

Comment:
use a public released image from cartesi instead of oap75

Reply:
Fixed !